server/test/auth/ProfileMergeTest.java

```diff
@@ -82,7 +83,7 @@ public void testProfileCreation() throws ExecutionException, InterruptedExceptio
     OidcProfile oidcProfile = createOidcProfile("foo@example.com", "issuer", "subject");
 
     CiviFormProfileData profileData =
-        idcsProfileAdapter.civiformProfileFromOidcProfile(oidcProfile);
+        idcsProfileAdapter.mergeCiviFormProfile(Optional.empty(), oidcProfile);
```

Comment:
inline doc the Optional.empty() added in this file similar to the other comment.